t/plugin/limit-count.t

```diff
@@ -1175,3 +1175,73 @@ passed
 [200, 200, 503, 503]
 --- no_error_log
 [error]
+
+
+
+=== TEST 37: add route and upstream is the domain name
```

Comment:
The #3108 is limit-count on the `service`, not a route?

Reply:
The main test is the case where the upstream is a domain name, which is also feasible.
But I am adding some tests that limit the service.